modules/custom/az_flexible_page/css/az_flexible_page.theme.css

```diff
@@ -2,7 +2,7 @@
 *  Page-specific overrides to the theme defaults.
 */
 
-.node--type-az-flexible-page .node__content {
+.node--type-az-flexible-page article.node--view-mode-full:first-child {
```

Comment:
This isn't specific enough for this page: https://live-azs-demo.pantheonsite.io/pages/accordions since the node__content class is overriding it.

Reply:
Try 

.node--type-az-flexible-page article.node--view-mode-full .node__content:nth-of-type(1)

Reply:
Any reason not to simply use the child combinator `>` & keep node__content in the rule?

Reply:
The flexible page with the list of flexible pages 

structured like 

-.node--type-az-flexible-page
--article.node--view-mode-full
--- .node__content
---- node__content
---- node__content
---- node__content

Reply:
Just checked that the https://live-azs-demo.pantheonsite.io/pages/accordions case was unfortunately indistinguishable from the one that should get the override (so much for that idea).